children/README.md

```diff
@@ -0,0 +1,156 @@
+# STAC API - Children
```

Comment:
Are catalogs returned by this endpoint allowed to implement item search (i.e. can they have a `/search` endpoint)?

Reply:
Yes, any (sub) Catalog can implement a search endpoint.

Reply:
Thanks for bringing this up. I thought I'd explicitly mentioned this, but I can't find it, so I'll make sure to add something.